### PR TITLE
tf2ify/melodify collision_velocity_filter

### PR DESCRIPTION
**Pull request by jabbenseth**
we use this pattern when necessary. it is backwards compatible to indigo...

### CODE REVIEW

File: cob_collision_velocity_filter/include/cob_collision_velocity_filter.h [hunk 1]
```diff
@@ -51,7 +51,12 @@
 #include "velocity_limited_marker.h"
 
 // Costmap for obstacle detection
+#if ROS_VERSION_MINIMUM(1, 14, 0)
+#include <tf2_ros/transform_listener.h>
```
Comment (fmessmer) [inline cob_collision_velocity_filter/include/cob_collision_velocity_filter.h]: we still would need a `<build_depend>tf2_ros</build_depend>` in the `package.xml`, right?

**Review by fmessmer — CHANGES_REQUESTED** [review]

File: cob_collision_velocity_filter/include/cob_collision_velocity_filter.h (hunk 1 above)
Comment (jabbenseth) [reply cob_collision_velocity_filter/include/cob_collision_velocity_filter.h]: see 8e1efb2

**Review by fmessmer — CHANGES_REQUESTED** [review]
please rebase to enable melodic tests in travis

File: cob_collision_velocity_filter/include/cob_collision_velocity_filter.h (hunk 1 above)
Comment (benmaidel) [reply cob_collision_velocity_filter/include/cob_collision_velocity_filter.h]: You could move the include to the cpp. It's just used there.

**Review by benmaidel — DISMISSED** [review]

**Comment by jabbenseth** [comment]
close in favour of #208

**Pull request closed without merging**